pkg/curatedpackages/packageclient.go

```diff
@@ -0,0 +1,48 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	packagesv1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+const (
+	minWidth = 16
+	tabWidth = 8
+	padding  = 0
+	padChar  = '\t'
+	flags    = 0
+)
+
+type PackageClient struct {
+	packages []packagesv1.BundlePackage
+}
+
+func NewPackageClient(packages []packagesv1.BundlePackage) *PackageClient {
+	return &PackageClient{
+		packages: packages,
+	}
+}
+
+func (p *PackageClient) DisplayPackages() {
+	w := new(tabwriter.Writer)
+	defer w.Flush()
+	w.Init(os.Stdout, minWidth, tabWidth, padding, padChar, flags)
+	fmt.Fprintf(w, "%s\t%s\t \n", "PackageClient", "Version(s)")
```

Comment:
How does kubectl do this? For example, the bars below would vary based on the length of the other strings right?

Reply:
I'd raised this previously. Have a read of the tabwriter pkg https://pkg.go.dev/text/tabwriter. 

> Package tabwriter implements a write filter (tabwriter.Writer) that translates tabbed columns in input into properly aligned text